pipes.lua

```diff
@@ -41,7 +41,7 @@ for index, connects in ipairs(cconnects) do
 	local pipedesc = S("Pipe Segment").." "..dump(connects)
 
 	if #connects == 0 then
-		pgroups = {snappy = 3, tube = 1, dig_generic = 4, axey=1, handy=1, pickaxey=1}
+		pgroups = {snappy = 3, pipe = 1, dig_generic = 4, axey=1, handy=1, pickaxey=1}
```

Comment:
what mechanisms rely on that group name anyway?

Reply:
oh sorry, that's part of another pull request that leaked over

Reply:
while editing that line, maybe also add spaces around '=' signs where missing...

Reply:
> while editing that line, maybe also add spaces around '=' signs where missing...

should be done now: https://github.com/mt-mods/pipeworks/pull/153/commits/d32af9bba3f8070f8f8425bc17b9f39e5cf2bbd8